README.md

```diff
@@ -235,6 +240,67 @@ To move a CR to a PR:
 See https://gist.github.com/trentm/6188f14cd7487ac288c85080bfa7c3f3 for what
 an example run looks like.
 
+## How to duplicate a PR
+
+As mentioned above, when moving a CR to a PR, the submitter of the PR is
+the user who performed the migration.
+
+If you need to re-assert ownership of a PR (important, because GitHub's
```

Comment:
Personally I'd change "need" to "want", and temper the "important". If the particular author doesn't want to bother because they feel the churn is more effort/distraction than getting the commit credit, then that's their choice.

Reply:
That's reasonable - I've made changes here.